Corona-Warn-App/src/main/res/values-bg/strings.xml

```diff
@@ -66,8 +66,6 @@
     <!-- NOTR -->
     <string name="preference_teletan"><xliff:g id="preference">"preference_teletan"</xliff:g></string>
     <!-- NOTR -->
-    <string name="preference_last_three_hours_from_server"><xliff:g id="preference">"preference_last_three_hours_from_server"</xliff:g></string>
-    <!-- NOTR -->
```

Comment:
Afaik, we usually only delete keys in DE and the Translation Delivery will remove the remaining ones.

Reply:
Ok will do next time.